// terra-doc-template/lib/terra-dev-site/doc/doc-template/Placeholder.2.doc.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _DocTemplate = require('../../../DocTemplate');

var _DocTemplate2 = _interopRequireDefault(_DocTemplate);

var _PLACEHOLDER = require('../../../../docs/PLACEHOLDER.md');

var _PLACEHOLDER2 = _interopRequireDefault(_PLACEHOLDER);

var _package = require('../../../../package.json');

var _Placeholder = require('!raw-loader!../../../../src/Placeholder.jsx');

var _Placeholder2 = _interopRequireDefault(_Placeholder);

var _PlaceholderDefault = require('../example/PlaceholderDefault');

var _PlaceholderDefault2 = _interopRequireDefault(_PlaceholderDefault);

var _PlaceholderDefault3 = require('!raw-loader!../../../../src/terra-dev-site/doc/example/PlaceholderDefault');

var _PlaceholderDefault4 = _interopRequireDefault(_PlaceholderDefault3);

var _PlaceholderLight = require('../example/PlaceholderLight');

var _PlaceholderLight2 = _interopRequireDefault(_PlaceholderLight);

var _PlaceholderLight3 = require('!raw-loader!../../../../src/terra-dev-site/doc/example/PlaceholderLight');

var _PlaceholderLight4 = _interopRequireDefault(_PlaceholderLight3);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

/* eslint-enable import/no-webpack-loader-syntax, import/first, import/extensions, import/no-unresolved, import/no-duplicates */

// Example Files
var DocPage = function DocPage() {
  return _react2.default.createElement(_DocTemplate2.default, {
    packageName: _package.name,
    readme: _PLACEHOLDER2.default,
    srcPath: 'https://github.com/cerner/terra-core/tree/master/packages/' + _package.name,
    examples: [{
      title: 'Dark Placeholder',
      example: _react2.default.createElement(_PlaceholderDefault2.default, null),
      source: _PlaceholderDefault4.default
    }, {
      title: 'Light Placeholder',
      example: _react2.default.createElement(_PlaceholderLight2.default, null),
      source: _PlaceholderLight4.default
    }],
    propsTables: [{
      componentName: 'PlaceholderSrc',
      componentSrc: _Placeholder2.default
    }]
  });
};

/* eslint-disable import/no-webpack-loader-syntax, import/first, import/extensions, import/no-unresolved, import/no-duplicates */
// Component Source
exports.default = DocPage;
